resource_estimator/src/estimates/data/result.rs

```diff
@@ -184,12 +184,12 @@ fn create_physical_resource_counts_breakdown<L: Overhead + Clone>(
         num_tstates: result
             .layout_overhead()
             .num_tstates(num_ts_per_rotation.unwrap_or_default()),
-        num_tfactories: result.num_tfactories(),
-        num_tfactory_runs: result.num_tfactory_runs(),
-        physical_qubits_for_tfactories: result.physical_qubits_for_tfactories(),
+        num_tfactories: result.num_factories(),
+        num_tfactory_runs: result.num_factory_runs(),
+        physical_qubits_for_tfactories: result.physical_qubits_for_factories(),
```

Comment:
same here. do we want to remove "t" from tfactories?

Reply:
Same comment as above.